ext_tables.php

```diff
@@ -5,6 +5,10 @@
     $GLOBALS['TYPO3_CONF_VARS']['BE']['toolbarItems'][1472569541] =
         \IchHabRecht\BegroupsRoles\Backend\ToolbarItems\RoleSwitcher::class;
 
+    // Flip order of RoleSwitch and UserToolbarItem
```

Comment:
Please add a comment here to where this "magic number" is coming from (sysext/backend/ext_localconf.php), so that it's not so "wtf" anymore for someone reading this code.

btw: Nice "API" that TYPO3 provides to add toolbarItems. :) The unix timestamps makes no sense at all in my opinion.